docs/guides/run_a_job.md

```diff
@@ -0,0 +1,227 @@
+    - [Mode](#mode): `--mode <mode_name>`
+    - [Environment](#environment): `--env <environment_name`
+    - [Message](#message) `--message` or `-m`
+    - [Tensorboard](#tensorboard): `--tensorboard`
+
+    **[[COMMAND]](#command_1)**
+
+Running jobs is the core action in the FloydHub workflow. A job pulls together
+your code and dataset(s), sends them to a deep-learning server configured with
+the right environment, and actually kicks off the necessary code to get the
+data science done. This article serves as a more in-depth look at the
+ins-and-outs of running jobs on FloydHub using the `floyd run` command.
+
+The `floyd run` command can be broken down into the two main parts: the
+`[OPTIONS]`, and the `[COMMAND]`. We'll detail all the `[OPTIONS]` available to
+you, as well as how to use the `[COMMAND]` properly. Use the links below as a
+quick reference.
+
+## Parts of the `floyd run` Command
+
+### [[OPTIONS]](#options_1)
+- [Instance Type](#instance-type): `--cpu --gpu --cpu+ --gpu+`
+- [Dataset(s)](#datasets): `--data`
+- [Mode](#mode): `--mode`
+- [Environment](#environment): `--env`
+- [Message](#message) `--message -m`
+- [Tensorboard](#tensorboard): `--tensorboard`
+
+### [[COMMAND]](#command_1)
+
+## [OPTIONS]
+### Instance Type
+
+To specify the instance type means to choose what kind of FloydHub instance
+your job will run on. Think of this as a hardware choice rather than a software
+one. (The software environment is declared with the
+[Environment (`--env`)](#environment) `OPTION` of `floyd run` command.)
+
+You have four instance type options to choose from when running a job as
+detailed below:
+
+|`floyd run` Flag|Instance Type|Description           |
+|:------------|:--------------:|:---------------------|
+|`--gpu`      |GPU             |Preemptible GPU server|
+|`--gpu+`     |GPU             |Dedicated GPU server  |
+|`--cpu`      |CPU             |Preemptible CPU server|
+|`--cpu+`     |CPU             |Dedicated CPU server  |
+
+!!! important
+
+    - The default instance flag is `--cpu`. This means that if you don't pass
+      any of the above flags to `floyd run`, your job will be run on a
+      preemptible CPU server.
+    - If you pass more than one instance flag, this is the order of precedence:
+      `--gpu+`, `--cpu+`, `--gpu`, `--cpu`
+
+#### What is the difference between a preemptible and a dedicated instance?
+
+Refer to [this section](../faqs/plans/#preemptible-vs-dedicated-instances)
+
+
+### Dataset(s)
+
+You can specify up to five datasources (datasets or outputs from previous jobs)
+to mount to the server that will be running your job. For each datasource,
+specify the `--data` flag as detailed below:
+
+```
+--data <name_of_datasource>:<mount_point_on_server>
+```
+
+For more detailed information on mounting data to jobs, see
+[this article](../guides/data/mounting_data)
+
+### Mode
+
+FloydHub jobs can currently be run in one of three modes:
+
+1. `--mode job`
```

Comment:
So this will be `--mode command`? This needs a CLI change I think. @houqp can comment.